src/common/transformations/src/transformations/transpose_sinking/ts_utils.cpp

```diff
@@ -112,11 +112,6 @@ void SwapFriendlyNames(const NodePtr& node1, const NodePtr& node2) {
     node1->set_friendly_name(node2_name);
 }
 
-void SwapNames(const NodePtr& node1, const NodePtr& node2) {
-    SwapFriendlyNames(node1, node2);
-    SwapOutputNames(node1->output(0), node2->output(0));
```

Comment:
SwapFriendlyNames and SwapOutputNames both removed.
RemoveTransposeConsumers now sets friendly names, but where and how set_names() called for tensor names?

Reply:
Good point, `replace` method do it 
![image](https://user-images.githubusercontent.com/21181928/235132507-4c5d4706-56a6-4c3d-8124-3bf9635cd3e2.png)
![image](https://user-images.githubusercontent.com/21181928/235132579-e414cb6d-3bc9-424b-b4f1-b7d4410bbaa5.png)